aiogram/fsm/strategy.py

```diff
@@ -7,18 +7,20 @@ class FSMStrategy(Enum):
     CHAT = auto()
     GLOBAL_USER = auto()
     USER_IN_TOPIC = auto()
-
+    CHAT_TOPIC = auto()
 
 def apply_strategy(
     strategy: FSMStrategy,
     chat_id: int,
-    user_id: int,
+    user_id: Optional[int] = None,
     thread_id: Optional[int] = None,
-) -> Tuple[int, int, Optional[int]]:
+) -> Tuple[int, Optional[int], Optional[int]]:
     if strategy == FSMStrategy.CHAT:
         return chat_id, chat_id, None
     if strategy == FSMStrategy.GLOBAL_USER:
         return user_id, user_id, None
     if strategy == FSMStrategy.USER_IN_TOPIC:
         return chat_id, user_id, thread_id
+    if strategy == FSMStrategy.CHAT_TOPIC:
+        return chat_id, None, thread_id
```

Comment:
You should use `chat_id, chat_id, thread_id` instead of `chat_id, None, thread_id`.
Your strategy is very similar to `FSMStrategy.CHAT`.